src/lucky_migrator/create_table_statement.cr

```diff
@@ -72,7 +73,7 @@ class LuckyMigrator::CreateTableStatement
     {% end %}
   end
 
-  def add_column(name, type : ColumnType.class | Float.class, optional = false, reference = nil, on_delete = :do_nothing, default : ColumnType | Float | Symbol | Nil = nil)
+  def add_column(name, type : ColumnType, optional = false, reference = nil, on_delete = :do_nothing, default : ColumnDefaultType | Float = nil)
```

Comment:
I couldn't add `| Float` to the `ColumnDefaultType` directly, but I'm allowed to use it here. Odd but it works! :)